packages/scaffolding-cli/templates/src/ssr/package.json

```diff
@@ -44,15 +44,16 @@
     "axios": "^0.19.2",
     "body-parser": "^1.19.0",
     "cacheable-response": "^2.0.7",
+    "contentful": "^7.14.4",
     "cross-env": "^6.0.3",
+    "date-fns": "^2.14.0",
     "express": "^4.17.1",
     "express-session": "^1.17.1",
     "helmet": "^3.21.1",
     "isomorphic-unfetch": "^3.0.0",
     "morgan": "^1.9.1",
-    "next": "~9.4",
-    "next-redux-saga": "^4.1.2",
-    "next-redux-wrapper": "^5.0.0",
+    "next": "^9.2",
```

Comment:
Lower version?

Reply:
^ will get the latest minor version under 9 right? I am using 9.4 only.